### PR TITLE
test: car interface fuzzy testing

### PR DESCRIPTION
**Pull request by maxime-desroches**
Found an index out of bounds error for the body carcontroller https://github.com/commaai/openpilot/pull/28539

need to add hypothesis to agnos

### CODE REVIEW

File: selfdrive/car/tests/test_car_interfaces.py [hunk 1]
```diff
@@ -9,11 +11,36 @@
 from selfdrive.car.car_helpers import interfaces
 from selfdrive.car.fingerprints import _FINGERPRINTS as FINGERPRINTS, all_known_cars
 
+def get_random_car_control():
```
Comment (adeebshihadeh) [inline selfdrive/car/tests/test_car_interfaces.py:14]: use the existing stuff and extend it if necessary https://github.com/commaai/openpilot/blob/master/selfdrive/test/process_replay/test_fuzzy.py

**Review by adeebshihadeh — COMMENTED** [review]
Can you make a separate PR to fix the body bug?

File: selfdrive/test/process_replay/fuzzy_generation.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import hypothesis.strategies as st
+import random
```
Comment (adeebshihadeh) [inline selfdrive/test/process_replay/fuzzy_generation.py:2]: let's move this to selfdrive/test/ and need to add to release/files_common

**Review by adeebshihadeh — APPROVED** [review]

File: selfdrive/test/process_replay/fuzzy_generation.py (hunk 2 above)
Comment (maxime-desroches) [reply selfdrive/test/process_replay/fuzzy_generation.py]: I will make a separate PR for the test_car_interface stuff since it needs an agnos update

**Pull request merged**